core/src/main/scala/ai/h2o/sparkling/frame/H2OFrame.scala

```diff
@@ -120,4 +123,24 @@ object H2OFrame extends RestCommunication {
       location = clusterNodes(sourceChunk.node_idx))
   }
 
+  def initializeFrame(conf: H2OConf, frameId: String, columns: Array[String]): InitializeFrameV3 = {
```

Comment:
NIT (I wouldn't do it know): It would be nice for these methods to be private/ package-private from some point so `H2OFrame` can be user-facing class, but we can do any cleanups after this change is done

Reply:
Good point!

Reply:
Will do it in a different PR, It would require to move `ExternalWriteConverterCtx` to the same package as `H2OFrame`.